LWM2M_DevCapMgmt-v1_0_1.xml

```diff
@@ -15,17 +15,17 @@ NORMATIVE INFORMATION
 
   Information about this file can be found in the latest revision of
 
-    OMA-TS-LWM2M_DevCapMngt-V1_0
+    OMA-TS-LWM2M_DevCapMgmt-V1_0_3
 
-  This is available at http://www.openmobilealliance.org/
+  This is available at https://www.omaspecworks.org/
 
   Send comments to https://github.com/OpenMobileAlliance/OMA_LwM2M_for_Developers/issues
 
 CHANGE HISTORY
 
 08032017 Status changed to Approved by TP, TP Ref # OMA-TP-2017-0021R01-INP_LWM2M_DevCapMgmt_1.0.2_for_Final_Approval_Notification
-16062020 Status changed to Approved by DMSE, Doc Ref # OMA-DM_SE-2020-0018-INP_LwM2M_DevCapMgmt_V1_0_3_RRP_for_Final_Approval
-	
+16062020 Status changed to Approved by DMSE, Doc Ref # OMA-DM&SE-2020-0018-INP_LWM2M_DevCapMgmt_V1_0_3_RRP_for_Final_Approval
```

Comment:
@jmudge in the past "&" created some issues. Shouldn't it be DMSE?

Reply:
Please see the following screenshot from the portal:

![DM SE_document_ref](https://user-images.githubusercontent.com/11298435/85006306-da3b2d00-b151-11ea-97e9-02315e0417f3.png)

The current reference in LWM2M_DevCapMgmt-v1_0_1.xml seems to be correct.

Reply:
With regards to &. Ok.